docs/modules/ROOT/partials/storage-ceph-remove-osd.adoc

```diff
@@ -1,42 +1,40 @@
-. Find the OSD on the node you want to replace
+. Tell Ceph to take OSDs on old nodes out of service and relocate data stored on them
```

Comment:
Maybe use the same trick as we're using in the mon partial to make the text more focused for each how-to:

```suggestion
. Tell Ceph to take the OSD(s) on the node(s) to {operation} out of service and relocate data stored on them
```

Reply:
```suggestion
. Tell Ceph to take the OSD(s) on the node(s) to {mon-operation} out of service and relocate data stored on them
```